src/renderer/dx/DxRenderer.cpp

```diff
@@ -1700,7 +1696,7 @@ try
     // NOTE: Line coordinates are centered within the line, so they need to be
     // offset by half the stroke width. For the start coordinate we add half
     // the stroke width, and for the end coordinate we subtract half the width.
-
+    const DxFontRenderData::LineMetrics _lineMetrics = _fontRenderData->GetLineMetrics();

```

Comment:
nit: I know that this variable name was chosen to reduce code deltas in this area, but it would be better if we dropped the `_`

Reply:
Makes sense. Fixed